setup.py

```diff
@@ -22,17 +22,30 @@ def write_version_py(filename=None):
         fh.close()
 
 
-write_version_py()
+cfd = os.path.join(
+    os.path.dirname(__file__),
+    'wrappers/CFD', 'version.py'
+)
+dem = os.path.join(
+    os.path.dirname(__file__),
+    'wrappers/DEM', 'version.py'
+)
+
+write_version_py(cfd)
+write_version_py(dem)
 
 setup(
-    name='simkratos',
+    name='wrappers',
```

Comment:
I would suggest that you use a more specific name for the kratos plugin (e.g. `simphony_kratos`). This name is used for the package folder inside the `site-package` and using `wrappers` is very generic and could conflict with other packages.